### ISSUE TITLE
International keyboard layouts: Convert "CmdOrCtrl+[" and "CmdOrCtrl+]" accelerators to localized shortcuts

### ISSUE THREAD

**Issue by Sukram21** [electron/electron#8994]
* Electron version: ^1.6.2
* Operating system: macOS

Menu items like "Back" and "Forward" are usually implemented via "CmdOrCtrl+[" and "CmdOrCtrl+]" accelerators as shortcuts, e.g. see https://github.com/jiahaog/nativefier/blob/9243f6689fe1cacc2311ee009bd96b705b32d8ad/app/src/components/menu/menu.js#L77

### Actual behavior
Unfortunately, the mentioned shortcuts **will never trigger** on e.g. German keyboards, as "[" and "]" are not directly reachable (see [German keyboard layout]( https://en.wikipedia.org/wiki/German_keyboard_layout))

### Expected behavior
As the shortcuts are otherwise completely useless for German keyboards (I image its similar with other keyboards), I would expect that these accelerators will automatically get adjusted to their localized `KeyboardEvent.code`equivalent. 

Localized equivalent means, keys which trigger the same `KeyboardEvent.code`. Example: `event.code === "BracketLeft"` is `true` for a "[" press on en-US keyboards, on de-DE keyboards, it is `true` for an "ü" press (also: "]" is code-equivalent to "+"). See this JSFiddle to try it out: https://jsfiddle.net/xeq9or42/ 

This already works when adding a keydown event listener in a webpage (see code below) - Can't we have something similar for Electron menu shortcuts?

To sum it up: On German keyboards, the menu item with accelerator "CmdOrCtrl+[" should be triggered when "CmdOrCtrl+ü" is pressed, the menu item with accelerator "CmdOrCtrl+]" should be triggered on "CmdOrCtrl+Plus" press.

----------

Sample code that works on a webpage: 
```js
document.addEventListener('keydown', function(event) {
  console.log(event)

  if (event.code === "BracketLeft") {
  	console.log("back")
  	history.back()
  } else if (event.code === "BracketRight") {
  	console.log("forward")
  	history.forward()
  }
})
```
